src/Gui/ViewProviderLink.cpp

```diff
@@ -1793,6 +1840,10 @@ void ViewProviderLink::updateData(const App::Property *prop) {
     return inherited::updateData(prop);
 }
 
+static inline bool canScale(const Base::Vector3d &v) {
+    return fabs(v.x)>1e-7 && fabs(v.y)>1e-7 && fabs(v.z)>1e-7;
```

Comment:
Do we have a constant that can be used here?

Reply:
OCC Precision::Confusion()` is not available here. Not sure if we have any constant defined for this.